apps/edxapp/vars/all/main.yml

```diff
@@ -6,7 +6,7 @@ edxapp_lms_host: "lms.{{ project_name}}.{{ domain_name }}"
 
 # -- edxapp (cms/lms)
 edxapp_image_name: "fundocker/edxapp"
-edxapp_image_tag: "ginkgo.1-1.0.6"
+edxapp_image_tag: "hawthorn.1rc3-1.0.2"
```

Comment:
hawthorn.1-1.0.0 is now available :taco:

Reply:
That's on the roadmap of this PR!